### PR TITLE
MGMT-15332: Adding same name for 2 manifests - UI shows error on all manifests

### PR DESCRIPTION
**Pull request by jgyselov**
https://issues.redhat.com/browse/MGMT-15332

In both screenshots, the manifests were called `manifest1.yaml`, `manifest2.yaml`, and `manifest2.yaml`.

**Before:**

image

**After:**

image

### CODE REVIEW

**Comment by jgyselov** [comment]
@jkilzi PTAL especially at the tests. Personally, I'm happy with the structure but not quite so sure about using the aliases.

File: libs/ui-lib/lib/ocm/components/clusterConfiguration/manifestsConfiguration/components/customManifestsValidationSchema.tsx [hunk 1]
```diff
@@ -13,45 +12,40 @@ const INCORRECT_FILENAME = 'Must have a yaml, yml or json extension and can not
...
-      return this.createError({
-        message: 'Unexpected error: Failed to get values to test uniqueness',
-      });
-    }
-    return values.length === setValues.size;
-  });
-};
+    const filePath = `${folder}_${filename}`;
+    const values = context.values.manifests.map(
+      (manifest) => `${manifest.folder}_${manifest.filename}`,
```
Comment (jkilzi) [inline libs/ui-lib/lib/ocm/components/clusterConfiguration/manifestsConfiguration/components/customManifestsValidationSchema.tsx:30]: I think that this has changed in #2277

Comment (ammont82) [reply libs/ui-lib/lib/ocm/components/clusterConfiguration/manifestsConfiguration/components/customManifestsValidationSchema.tsx:30]: Yes, this has been changed in #2277

Comment (jgyselov) [reply libs/ui-lib/lib/ocm/components/clusterConfiguration/manifestsConfiguration/components/customManifestsValidationSchema.tsx:30]: Don't worry, I'll do a rebase. :slightly_smiling_face:

**Review by ammont82 — DISMISSED** [review]

File: libs/ui-lib/lib/ocm/components/clusterConfiguration/manifestsConfiguration/components/customManifestsValidationSchema.tsx (hunk 1 above)
Comment (ammont82) [reply libs/ui-lib/lib/ocm/components/clusterConfiguration/manifestsConfiguration/components/customManifestsValidationSchema.tsx:30]: @jgyselov resolve the conflicts

**Review by ammont82 — APPROVED** [review]

**Review by jkilzi — APPROVED** [review]

**Comment by jkilzi** [comment]
please rebase

**Pull request merged**